### PR TITLE
Issue94 link pull request

### PR DESCRIPTION
**Pull request by htsungwing**
It's impossible to auto merge because we were working from a different originated branch. Just making a pull request because Dana said so tee hee.

### CODE REVIEW

File: src/searchTab/trace-search-history.js [hunk 1]
```diff
@@ -41,10 +41,11 @@ export class TraceSearchHistory {
             
             this.firebase.on('value', function(snapshot) {
             let data = snapshot.val();
-            console.info(`${data.searchFilterId} inside subscribe firebase`);
-            console.info(`${data.searchTermText} inside subscribe firebase`);
+            // console.info(`${data.searchFilterId} inside subscribe firebase`);
```
Comment (jbhilton11) [inline src/searchTab/trace-search-history.js]: Remove commented code

File: src/searchTab/trace-search-history.js [hunk 2]
```diff
@@ -41,10 +41,11 @@ export class TraceSearchHistory {
...
+            // console.info(`${data.searchTermText} inside subscribe firebase`);
```
Comment (jbhilton11) [inline src/searchTab/trace-search-history.js]: Remove commented code

File: src/searchTab/trace-search-history.js [hunk 3]
```diff
@@ -54,8 +55,8 @@ export class TraceSearchHistory {
         this.eventAggregator.subscribe(searchBoxChangedEvent, payload => {
             let searchTermText = payload.searchTermText;
             let searchFilterId = payload.searchFilterId;
-            console.info(`${searchTermText} inside subscribe searchBoxChangedEvent`);
-            console.info(`${searchFilterId} inside subscribe searchBoxChangedEvent`);
+            // console.info(`${searchTermText} inside subscribe searchBoxChangedEvent`);
```
Comment (jbhilton11) [inline src/searchTab/trace-search-history.js]: Remove commented code

File: src/searchTab/trace-search-history.js [hunk 4]
```diff
@@ -54,8 +55,8 @@ export class TraceSearchHistory {
...
+            // console.info(`${searchFilterId} inside subscribe searchBoxChangedEvent`);
```
Comment (jbhilton11) [inline src/searchTab/trace-search-history.js]: Remove commented code

File: src/searchTab/trace-search.js [hunk 5]
```diff
@@ -2,184 +2,158 @@
...
+        this.publishTraceSearchChanged(this.searchedValue, this.selectedFilter);
+    }
+    // jumps to current line in the editor
+    doOnClickJumpToCode(row) {
+        let editor = ace.edit('aceJsEditorDiv');
+        editor.gotoLine(this.rows[row.$index].range.start.row+1);
+        
+    }
+    // highlights current line in the ditor
+    doOnClickHighlight(row){
```
Comment (Alaboudi1) [inline src/searchTab/trace-search.js]: So, basically any expression in the table that the user clicks on get highlighted until the user clicks on it again to remove the highlighting. I would say there would be a bug here if one expression is in multiple rows. For example, for loop. Have you guys thought about it? Another question, what is the use case here? Why would  the user want to highlight multiple expressions from the table?

**Comment by jbhilton11** [comment]
I will try to review code a little more, however, this pull request needs to resolve conflicts before this can be successfully merged. Please pull the latest master branch into your branch and resolve the conflicts.

**Comment by jbhilton11** [comment]
Please merge the latest master into your branch, resolve conflicts and push back up to Github for merging into master.

**Comment by luminaxster** [comment]
Fixed conflicts

**Pull request merged**